common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -676,16 +680,37 @@ class @Problem
       element = $(element)
       element.find("section[id^='forinput']").removeClass('choicetextgroup_show_correct')
 
+  disableAllButtonsWhileRunning: (buttonsHandlerCallback ,processCallback) =>
```

Comment:
This function is unnessarily abstracted. `buttonsHandlerCallback` will only ever be `enableAllButtons`, so I think this function only needs to take the parameter `processCallback`.

By the way, why are you calling it `processCallback`? Isn't `operation` more descriptive, like how https://github.com/edx/edx-platform/blob/master/common/static/common/js/components/utils/view_utils.js#L116-L122 does it?